{"uuid":"dba44e5c-fe56-4296-ab21-c793e748679e","name":"Scenario Outline: The user getting error message with invalid python code for Types of Linked List","children":["2986c75a-c7b5-4004-8074-1d074090ac76"],"befores":[],"afters":[{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696885693887,"stop":1696885693887},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696885694096,"stop":1696885694097},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696885694150,"stop":1696885694150},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696885696199,"stop":1696885696199}],"start":1696885693695,"stop":1696885696200}